### DISCUSSION TITLE
Balance non replicated classic queues

### DISCUSSION THREAD

**Discussion by MonicaMagoniCom** [rabbitmq/rabbitmq-server#7220] [Other]
Hi,

in our Kubernetes cluster, we have a RabbitMQ cluster where we have enabled classic queues without replication (in our scenario is not needed to have persistency) but I have encountered the following problem:
when a replica of the cluster is restarted (for any reason), then all the queues got concentrated on all the other replicas. Is not possible to rebalance the queues?

As far as I know, the command:
rabbitmq-queues rebalance all

does not work for classic queue non replicated.

**Comment by michaelklishin** [comment]
@dcorbacho do you remember if it does?

There is a different rebalancing tool in the support tools repo.
